README.md

```diff
@@ -59,14 +59,30 @@ steps:
 ```yml
 steps:
   - name: Collect Coverage
-    uses: tangro/actions-coverage@1.0.0
+    uses: tangro/actions-coverage@1.1.0
     with:
       post-comment: true
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       GITHUB_CONTEXT: ${{ toJson(github) }}
 ```
 
+There is also an option to adapt the minimum coverage percentage limits that need to be achived for the build to complete successfully (Defaults to `100`). You can configure an overall minimum by setting the environment variable `COVERAGE` and/or setting specific minimums for the different coverage metrics: `COVERAGE_LINES`, `COVERAGE_STATEMENTS`, `COVERAGE_FUNCTIONS` or `COVERAGE_BRANCHES`.
+
+## Example with alternate coverage minimums
+
+```yml
+steps:
+  - name: Collect Coverage
+    uses: tangro/actions-coverage@1.1.0
+    env:
+      COVERAGE: 94
```

Comment:
I would have preferred for the configuration values to be [parameters](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswith) but am fine with using environment variables.

Reply:
that does make sense - my first approach was to use environment variables but I agree, parameters are a better fit for these settings